justfile

```diff
@@ -120,7 +120,7 @@ euclid-wasm features='dummy_connector':
 precommit: fmt clippy
 
 # Check compilation of v2 feature on base dependencies
-v2_intermediate_features := "merchant_account_v2,payment_v2,customer_v2"
+v2_intermediate_features := "merchant_account_v2,payment_v2,customer_v2,business_profile_v2"
 hack_v2:
     cargo hack clippy --feature-powerset --ignore-unknown-features --at-least-one-of "v2 " --include-features "v2" --include-features {{ v2_intermediate_features }} --package "hyperswitch_domain_models" --package "diesel_models" --package "api_models"
```

Comment:
can we not have the feature-set check, we can check each create with independent v2 features alone. Can be taken up in a separate PR which includes only this change